en/concept/mixing.md

```diff
@@ -236,3 +236,33 @@ Idlespeed can range from 0.0 to 1.0. Idlespeed is relative to the maximum speed
 
 In the case where an actuator saturates, all actuator values are rescaled so that 
 the saturating actuator is limited to 1.0.
+
+#### Helicopter Mixer
+
+The helicopter mixer combines three control inputs (roll, pitch, thrust) into four outputs ( swash-plate servos and main motor ESC setting). The first output of the helicopter mixer is the throttle setting for the main motor. The subsequent outputs are the swash-plate servos. The tail-rotor can be controlled by adding a simple mixer.
+
+The thrust control input is used for both the main motor setting as well as the collective pitch for the swash-plate. It uses a throttle-curve and a pitch-curve, both consisting of five points. This allows for more complex behavior.
+
+The mixer definition begins with:
+
+```
+H: <number of swash-plate servos, either 3 or 4>
+T: <throttle setting at 0% thrust> <25%> <50%> <75%> <100%>
+P: <collective pitch at 0% thrust> <25%> <50%> <75%> <100%>
+```
+This is followed by either 3 or 4 lines for the swash-plate servos in the following form:
```

Comment:
This is followed by lines for each of the swash-plate servos (either 3 or 4) in the following form: